build.gradle.kts

```diff
@@ -9,13 +9,26 @@ configurations {
 
 dependencies {
   // Spring boot dependencies
+  implementation("org.springframework.boot:spring-boot-starter-security")
```

Comment:
Quite a few other dependencies introduced here.  Do we need them all?

Would prefer to add as and when we need them 🤷

Reply:
It was just in passing ... realised we'd need them soon, but can take out if you prefer?